scripts/tasks.js

```diff
@@ -10,20 +12,158 @@ exports.replaceWebpack = () => {
     to: 'node_modules/webpack/hot/log-apply-result.js'
   }];
 
-  replaceTasks.forEach(task => cp(task.from, task.to));
+  replaceTasks.forEach(task => shell.cp(task.from, task.to));
 };
 
+
+// The following implementation of a merge is written purely to be bug
+// free not to be optimized performance wise as this is not needed in this
+// case as it's just a dev script and the manifest files are very small
+// (relative to computation abilities) json files
+
+function isObject(candidate) {
+  return (typeof candidate) === 'object' && candidate !== null;
+}
+
+function isArray(candidate) {
+  return candidate.constructor === Array;
+}
+
+function mergeError(msg) {
+  throw new Error(`mergeManifests Error: ${msg}`);
+}
+
+function mergeArrays(passedArrA, passedArrB) {
+  // Deep copy both arrays just to be safe
+  const arrA = JSON.parse(JSON.stringify(passedArrA));
+  const arrB = JSON.parse(JSON.stringify(passedArrB));
+  // We never recurse on arrays as we assume each object
+  // in an array is self-contained and would not be broken up
+  // So don't assume this when modularizing the manifest files
+  const ret = _.uniq(arrA.concat(arrB));
+  // We use _.uniq which will remove duplicate strings and duplicate
+  // numbers but keep all objects as it would compare just their pointers
+  return ret;
+}
+
+function mergeObjects(passedObjA, passedObjB, manifestTypeA, manifestTypeB) {
+  // Prepare error origin text dependent on whether or not
+  // objects or strings were passed for original manifest merge
+  let originErrorText = '';
+  if ((typeof manifestTypeA) === 'string') {
+    originErrorText += ` in ${manifestTypeA}`;
+  }
+  if ((typeof manifestTypeB) === 'string') {
+    if (originErrorText) {
+      originErrorText += ` and ${manifestTypeB}`;
+    } else {
+      originErrorText += ` in ${manifestTypeB}`;
+    }
+  }
+
+  // Deep copy both objects just to be safe
+  const objA = JSON.parse(JSON.stringify(passedObjA));
+  const ret = JSON.parse(JSON.stringify(passedObjB));
```

Comment:
Yeah no problem, readability for sure. This is just a competitive programming habit haha ;). Everyone uses ret in that community.